### DISCUSSION TITLE
ATT&CK/Playbook Search Functionality looks to be broken in a fresh install of 2.3.21

### DISCUSSION THREAD

**Discussion by stummasino-matc** [Security-Onion-Solutions/securityonion#2759] [Unsupported Versions]
> In the ATT&CK navigator "selection controls -> search" not working  
So it looks like it is working but a little different that I thought it did. When you initially open up the search all of the "properties searched" are selected by default. The attack ID seems to break something when it is selected (unless you select it alone and then search for a specific attack ID). 

The linking between playbook and the ATT&CK navigator also seems to be broken. 
- Setting a Play to active does not show up in the ATTACK navigator as the docs indicate it should.

- When right clicking on a technique in the ATT&CK navigator and selecting "view related plays" it always comes back with "No data to display". (I am checking techniques that do have a related play in playbook that should match). 

Did I set something up wrong or is this a known issue?

**Comment by weslambert** [comment]
I'll need to check on this, but I believe this is a known issue.

**Reply by defensivedepth** [reply]
Yep, this is in progress for a fix
